main.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/openshift/assisted-image-service/pkg/servers"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/sirupsen/logrus"
 	log "github.com/sirupsen/logrus"
```

Comment:
Can we deduplicate this in any sane way? Maybe its better to not use a global logger or would that be too much change for this PR?

Reply:
Ah, I think that's just an oversight on my part, not noticing how the IDE added another import for it. Should be easy to get rid of the extra import.